kbase-extension/static/kbase/css/kbaseNarrative.css

```diff
@@ -2463,11 +2463,11 @@ button.kb-data-obj {
 
 .kb-data-list-add-data-button:hover {
     cursor:pointer;
-    background-color: #DC3C31;
+    background-color: #4379B1;
```

Comment:
Should the colors modify slightly on hover? I don't think that was specified in the ticket. If not, then this line could be removed.

Reply:
Pinged the Andy and Mallory to get an answer on this :)

Reply:
changed to the correct hover color